src/pages/projects/TrendingProjectCard.tsx

```diff
@@ -101,117 +101,122 @@ export default function TrendingProjectCard({
 
   return (
     <Link
-      style={{
-        borderRadius: radii.lg,
-        cursor: 'pointer',
-        overflow: 'hidden',
-      }}
       key={project.handle}
-      to={
+      href={
         project.cv === '2' ? v2ProjectRoute(project) : `/p/${project?.handle}`
       }
     >
-      <div style={cardStyle} className="clickable-border">
-        <div style={{ marginRight: 20, display: 'flex', alignItems: 'center' }}>
-          <div style={rankStyle}>{rank}</div>
-          <ProjectLogo
-            uri={metadata?.logoUri}
-            name={metadata?.name}
-            size={size === 'sm' ? 70 : 110}
-          />
-        </div>
+      <div
+        style={{
+          borderRadius: radii.lg,
+          cursor: 'pointer',
+          overflow: 'hidden',
```

Comment:
**question** i saw this above too. Would you mind explaining why it's needed?

Reply:
NextJS Link is different to `react-router-dom`. I think react behaves more like `<a href` whereas `Link` is more a logical container, so you need to supply your own visual container logic inside of it.

https://nextjs.org/docs/migrating/from-react-router

Reply:
fair call fair call